script_tests/tests/1_test-transfer.js

```diff
@@ -83,4 +83,57 @@ describe("Test Transfer Balance", function () {
 	it("Get nonce after transfer balance 2", async function () {
 		expect(await web3.eth.getTransactionCount(addressFrom, "latest")).to.eq(2);
 	});
+
+	it("Transfer balance 3", async function () {
+		const createTransaction = await web3.eth.accounts.signTransaction(
+			{
+				from: addressFrom,
+				to: addressTo,
+				value: web3.utils.toWei("50", "ether"),
+				gas: "5000000000",
+			},
+			privKey
+		);
+
+		const createReceipt = await web3.eth.sendSignedTransaction(
+			createTransaction.rawTransaction
+		);
+
+		expect(createReceipt.transactionHash).to.be.equal(
+			"0x55bfc4e08a28b386b00f1f43e3ece364b6fc23d10265ad565fab83f7b0b0782f"
+		);
+	}).timeout(10000);
+
+	it("Get accounts balance after transfer balance 3", async function () {
+		const balanceFrom = web3.utils.fromWei(await web3.eth.getBalance(addressFrom), "ether");
+		const balanceTo = await web3.utils.fromWei(await web3.eth.getBalance(addressTo), "ether");
+
+		expect(balanceFrom).to.be.equal("63.45678899999999999");
+		expect(balanceTo).to.be.equal("60");
+	});
+
+	it("Withdraw value from sender", async function () {
+		// target address = "723908ee9dc8e509d4b93251bd57f68c09bd9d04471c193fabd8f26c54284a4b(5EeUFyFjHsCJB8TaGXi1PkMgqkxMctcxw8hvfmNdCYGC76xj)";
+		// value = 30
```

Comment:
```
$ seth sig "withdraw(bytes32,uint256)"  
0x040cf020
```
If there is a function selector, the encoding is different.
```shell
$ seth --to-uint256 30
0x000000000000000000000000000000000000000000000000000000000000001e
$ seth --to-bytes32 1e
0x1e00000000000000000000000000000000000000000000000000000000000000
```

Reply:
If I understand correctly, do you mean that the input test data should not be hardcoded here and should be generated by the tool?

Reply:
I mean that If there is a function selector `f(bytes32,uint256)`,  `input` should be encoded by that way.

Reply:
@hujw77 Didn't get it.

`withdraw(bytes32,uint256)` is a wrapper contract in the description, it is not used here.

Reply:
I mean that `value` is encoded by bytes32 not uint256,  It should be more clearly.

Reply:
By "32 bits for the withdrawal amount", @AsceticBear what't the encoded type for it?

Reply:
U256 little-endian.